packages/dd-trace/src/config.js

```diff
@@ -743,6 +349,8 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       this._applyRemote(options)
     } else {
       this._applyOptions(options)
+      this._applyEnvironment()
+      this._applyCalculated()
```

Comment:
This is inconsistent with the constructor. Although it shouldn't matter as they work independently, having the same order will avoid confusion.

Reply:
Also, why is it needed to reapply those when the programmatic config changes, but not when the remote config changes?